### PR TITLE
update README

### PR DESCRIPTION
**Pull request by rhagarty**
Initial pass

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,134 +1,122 @@
 # Predicting flight delay and building an ML pipeline using R4ML
 
-In this developer journey we will use R4ML, a scalable R package, running on IBM Data Science Experience (DSX) to perform various Machine Learning exercises. For those users who are unfamiliar with the Data Science Experience, DSX is an interactive, collaborative, cloud-based environment where data scientists, developers, and others interested in data science can use tools (e.g., RStudio, Jupyter Notebooks, Spark, etc.) to collaborate, share, and gather insight from their data.
+In this developer journey we will use R4ML, a scalable R package, running on IBM Watson Studio to perform various Machine Learning exercises. For those users who are unfamiliar with the Watson Studio, it is an interactive, collaborative, cloud-based environment where data scientists, developers, and others interested in data science can use tools (e.g., RStudio, Jupyter Notebooks, Spark, etc.) to collaborate, share, and gather insight from their data.
```
Comment (stevemar) [inline README.md]: remove "the" from "the Watson Studio"

File: README.md [hunk 2]
```diff
@@ -1,134 +1,122 @@
...
 
 When the reader has completed this journey, they will understand how to:
 
 * Use [Jupyter Notebooks](http://jupyter.org/) to load, visualize, and analyze data.
-* Run Notebooks in [IBM Data Science Experience](https://datascience.ibm.com/).
-* Leverage [R4ML](https://github.com/SparkTC/r4ml) to conduct preprocessing and exploratory analysis with big data
+* Run Notebooks in [IBM Watson Studio](https://dataplatform.ibm.com).
+* Leverage [R4ML](https://github.com/SparkTC/r4ml) to conduct preprocessing and exploratory analysis with big data.
```
Comment (stevemar) [inline README.md]: this repo was moved to https://github.com/CODAIT/r4ml

File: README.md [hunk 3]
```diff
@@ -1,134 +1,122 @@
...
 
 The Intended audience of this code pattern is data scientists, who wish to apply scalable machine learning algorithms using R.
-R4ML provides various out of the box algorithms to experiments with. This specific Code Pattern will provide a SVM (Suport Vector Machine) example to demonstrate the ease and power of R4ML in implementing the scalable classification. For more information about additional functionality support, documentation, and the roadmap, please vist [R4ML](https://github.com/SparkTC/r4ml)
 
+This specific Code Pattern will provide a SVM (Suport Vector Machine) example to demonstrate the ease and power of R4ML in implementing scalable classification. R4ML provides various out-of-the-box algorithms to experiment with. For more information about additional R4ML functionality, support, documentation, and roadmap, please vist [R4ML](https://github.com/SparkTC/r4ml).
```
Comment (stevemar) [inline README.md]: the repo moved https://github.com/CODAIT/r4ml

File: README.md [hunk 4]
```diff
@@ -1,134 +1,122 @@
...
 
- - For this example, we will use, a subset of above dataset, which is shipped with R4ML
-
- - User can use the bigger dataset from RITA and our code will work with that.
+* Learn Large Scale Model Training for classification using a Support Vector Machine.
+* Learn Large Scale Model Tuning using Cross validation.
 
+### Source of data
 
+- We will use the Airline On-Time Statistics and Delay Causes from [RITA](http://www.rita.dot.gov). A 1% sample of the "airline" dataset is available [here](http://stat-computing.org/dataexpo/2009/the-data.html). All of the data is in the public domain.
```
Comment (stevemar) [inline README.md]: nit, use `*` for bullets

Comment (stevemar) [reply README.md]: why is "airline" in quotes? just remove it and say dataset

Comment (stevemar) [reply README.md]: nit, replace "here" with "by the American Statistical Association (ASA)"

File: README.md [hunk 5]
```diff
@@ -1,134 +1,122 @@
...
 * [IBM Analytics for Apache Spark](https://console.ng.bluemix.net/catalog/services/apache-spark): An open source cluster computing framework optimized for extremely fast and large scale data processing.
-* [Bluemix Object Storage](https://console.bluemix.net/catalog/services/object-storage): Build and deliver cost effective apps and services with high reliability and fast speed to market in an unstructured cloud data store.
+* [IBM Cloud Object Storage](https://console.bluemix.net/catalog/services/object-storage): Build and deliver cost effective apps and services with high reliability and fast speed to market in an unstructured cloud data store.
 
 ## Featured Technologies:
 
 * [Jupyter Notebooks](http://jupyter.org/): An open source web application that allows you to create and share documents that contain live code, equations, visualizations and explanatory text.
 * [R4ML](https://github.com/SparkTC/r4ml): R4ML is a scalable, hybrid approach to ML/Stats using R, Apache SystemML, and Apache Spark
```
Comment (stevemar) [inline README.md]: repo again here

File: README.md [hunk 6]
```diff
@@ -180,6 +168,46 @@ options to specify exactly what you want shared from your notebook:
...
+* Provides a collection of canned algorithms.
+
+* Provides the ability to create custom ML algorithms.
+
+* Provides both SparkR and Apache SystemML functionality.
+
+* APIs that should be familiar to R users.
+
+## Sample output
+
```
Comment (stevemar) [inline README.md]: this is missing

File: README.md [hunk 7]
```diff
@@ -180,6 +168,46 @@ options to specify exactly what you want shared from your notebook:
...
+## Links
+
+ - Watson Studio: https://datascience.ibm.com/docs/content/analyze-data/creating-notebooks.html.
```
Comment (stevemar) [inline README.md]: use `*` for bullets

Comment (stevemar) [reply README.md]: use actual hyperlink format

Comment (stevemar) [reply README.md]: Include R4ML, i guess

File: README.md [hunk 8]
```diff
@@ -180,6 +168,46 @@ options to specify exactly what you want shared from your notebook:
 * `All content, including code`: displays the notebook as is.
 * A variety of `download as` options are also available in the menu.
 
+## Explore and Analyze the Data
+
+The notebook is well documented and will guide you through the exercise. Some of the main tasks that will be covered include:
+
+* Load packages and data and do the initial transformation and various feature engineering.
```
Comment (stevemar) [inline README.md:175]: remove the new lines between the bullets

**Review by stevemar — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (rhagarty) [reply README.md]: Just a placeholder for now

**Comment by rhagarty** [comment]
Changes added

**Review by stevemar — APPROVED** [review]

**Pull request merged**